cmd/flex/main/cli.go

```diff
@@ -626,13 +626,13 @@ func readConfig(configFile string) (resources.UbiquityPluginConfig, error) {
 }
 
 func setupLogger(logPath string) (*log.Logger, *os.File) {
-	logFile, err := os.OpenFile(path.Join(logPath, k8sresources.UbiquityFlexLogFileName), os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0640)
+	logFile, err := os.OpenFile(path.Join(logPath, k8sresources.UbiquityFlexLogFileName), os.O_WRONLY|os.O_CREATE, 0640)
 	if err != nil {
```

Comment:
why?
flex should write logging to a file
so if u overwrite u will not have valide logs for flex, becausrle flex is a cli that triggered for attach detach.... so overwrite the file is not good.

Reply:
So we don't need to change the flex logs, still out to the files, abandon this change